src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsinterventionsservice/dto/InterventionDTO.kt

```diff
@@ -6,17 +6,18 @@ import java.util.UUID
 
 data class InterventionDTO(
   val id: UUID? = null,
-  val name: String? = null,
-  val createdAt: OffsetDateTime? = null
-
+  val title: String? = null,
+  val createdAt: OffsetDateTime? = null,
+  val regionList: String,
```

Comment:
i think this would be better as an actual `List<String>`

Reply:
I agree - it was initially a String to help with the mapping from csv.  I didn't quite get this working (as a list) so reverted back....I'll swap back as that is definitely what we want.